runtime/pom.xml

```diff
@@ -15,10 +15,6 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-core</artifactId>
     </dependency>
-    <dependency>
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-rest-client-jackson</artifactId>
-    </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-qute</artifactId>
```

Comment:
I would recommend to set the scope of ``quarkus-oidc-client-filter`` from ``provided`` to ``test``.  The dependency is required by test classes only and at runtime the app might define a ``reactive oidc filter`` dependency. And a comment explaining why a classic dependency is required would help as well (see my PR)

Reply:
Actually, there are several non-test classes that use it. And that's a problem. I think I'll need to create a reactive and a classic version for each of these classes.

Reply:
Good catch!
Done. I had to add a couple of dependencies that we lose when we reduce the scope of `quarkus-oidc-client-filter`.